packages/engine/lib/error/src/hook.rs

```diff
@@ -110,3 +110,12 @@ impl Report {
         DISPLAY_HOOK.get()
     }
 }
+
+impl<T> Report<T> {
+    /// Converts the `&Report<Context>` to `&Report<()>` without modifying the frame stack.
```

Comment:
Should we add a bit of the reasoning of why this exists? Perhaps we can even be explicit that this is purposefully only `pub(crate)`